featuretools/primitives/standard/transform_primitive.py

```diff
@@ -853,6 +853,7 @@ class EmailAddressToDomain(TransformPrimitive):
         >>> email_address_to_domain(['name@gmail.com', 'name@featuretools.com']).tolist()
         ['gmail.com', 'featuretools.com']
     """
+    name = "email_address_to_domain"
```

Comment:
Without testing - this makes me wonder if we also need to add the `compatibility` attribute to this and the other new primitives? Are they all compatible with all three dataframe libraries?

Reply:
for now they should inherit pandas compatibility from PrimitiveBase

made #1549 to track compatibility testing